index.html

```diff
@@ -227,6 +227,125 @@ <h3 class="app__text">
         </div>
       </div>
     </div>
+    <section class="reviews">
+      <div class="container__main">
+        <h2 class="reviews__title">Guests reviews</h2>
+        <div class="reviews__inner">
+          <svg class="reviews__arrow">
+            <use xlink:href="./sprite.svg#arrow"></use>
+          </svg>
+          <div class="reviews__item">
+            <img class="reviews__icon" src="img/bali.jpg" alt="bali" />
+            <div class="reviews__icon--text">From 250 USD</div>
+            <div class="reviews__name">
+              <p>Ubud Bali Resort&SPA</p>
+              <svg class="reviews__svg--name">
```

Comment:
You shouldn't use image for such type of info, in real life you will take such information from api
Please use css for styles instead of image